manifests/charts/istio-cni/templates/daemonset.yaml

```diff
@@ -92,6 +92,8 @@ spec:
               path: /readyz
               port: 8000
           securityContext:
+            appArmorProfile:
```

Comment:
is this the default value? if not, what about make it optional by `with values.appArmorProfile` or something else?

Reply:
1. It is not the default value.
2. Because `istio-cni` is privileged, it is the only value that `istio-cni` can work with on systems with appArmor.
3. It is a no-op on any platform without appArmor.
4. So this value will work everywhere.
5. We have no reason given the above to expose and document it as a toggle to users, but if someone later comes up with a real need to do that, it would be easy enough to do at that time.

Reply:
what I'm thinking is that helm template will generated same manifests output with same `values.yaml` across different versions.